tagging/tagging.py

```diff
@@ -29,6 +30,12 @@ async def tag(self, ctx: commands.Context, tag: Optional[str]) -> None:
         else:
             name = clean_name
 
+        embed = discord.Embed(
+            title=f"Tag `{tag}` applied successfully",
+            description="Changes may take up to 10 minutes to take effect.",
+            color=discord.Color.green(),
+        )
+        await ctx.send(embed=embed)
         await ctx.channel.edit(name=name)
         await ctx.message.add_reaction("\u2705")
```

Comment:
```suggestion
        await ctx.reply("\u2705 Changes may take up to 10 minutes to take effect due to rate-limits.")
        await ctx.channel.edit(name=name)
```

Having them embed say it was successful in the title, and then say it make take 10 minutes in the description seemed odd. 

This makes it clear to new mods why it would take 10 minutes, and also removes the need to add the reaction afterwards too.

Reply:
Will do, Chris. I'd like to keep the reaction though. It will be added only after the channel name has been changed.

Reply:
https://github.com/python-discord/modmail-plugins/commit/db6b4bfc8036a75a51c65b79552aaa765169f106 🤗